### PR TITLE
Code Quality: Added license header to XAML

### PR DESCRIPTION
**Pull request by 0x5bfa**
### Description

Add license header

```xaml
<!--  Copyright (c) 2023 Files Community. Licensed under the MIT License. See the LICENSE.  -->
```

### PR Checklist

- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
  - Related #11853 
- [x] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [ ] Are there any other steps that were used to validate these changes?

### Screenshots

_None_

### CODE REVIEW

File: src/Files.App/Properties/PublishProfiles/win10-arm64.pubxml
Comment (yaira2) [inline src/Files.App/Properties/PublishProfiles/win10-arm64.pubxml:1]: Do we want to add the header to this file? It's just a config file.

Comment (0x5bfa) [reply src/Files.App/Properties/PublishProfiles/win10-arm64.pubxml:1]: But we can.
@hishitetsu What do you think?

Comment (hishitetsu) [reply src/Files.App/Properties/PublishProfiles/win10-arm64.pubxml:1]: I wouldn't add it, but I also don't think it's wrong to add it.
In other words, it is up to the project maintainer.

Comment (yaira2) [reply src/Files.App/Properties/PublishProfiles/win10-arm64.pubxml:1]: Let's remove it

**Comment by 0x5bfa** [comment]
@yair100 Ready for review

File: src/Files.App/Properties/PublishProfiles/win10-arm64.pubxml
Comment (yaira2) [inline src/Files.App/Properties/PublishProfiles/win10-arm64.pubxml:1]: Can you revert the changes to the file? This shouldn't be showing up in the list of changed files.

Comment (0x5bfa) [reply src/Files.App/Properties/PublishProfiles/win10-arm64.pubxml:1]: Done

**Review by yair100 — APPROVED** [review]
LGTM
